web-ui/view/github.ml

```diff
@@ -34,19 +37,14 @@ let github_pr_url ~org ~repo id =
 
 let format_org org = li [ a ~a:[ a_href (org_url org) ] [ txt org ] ]
 
-let format_repo ~org { Client.Org.name; master_status } =
-  li
-    ~a:[ a_class [ Build_status.class_name master_status ] ]
-    [ a ~a:[ a_href (repo_url org name) ] [ txt name ] ]
+(* let ref_name r =
+   match Astring.String.cuts ~sep:"/" r with
+   | "refs" :: "heads" :: branch -> Astring.String.concat ~sep:"/" branch
+   | [ "refs"; "pull"; id; "head" ] -> id
+   | _ -> Fmt.str "Bad ref format %S" r *)
```

Comment:
If it's not needed anymore you could remove it: we have git versioning if we want to go back :wink: